core/unit_test/tstCommunicationPlan.hpp

```diff
@@ -575,6 +575,12 @@ void test7( const bool use_topology )
         EXPECT_EQ( n, host_steering( n ) );
 }
 
+void test_copy()
+{
+    Cabana::CommunicationPlan<TEST_DEVICE> comm_plan( MPI_COMM_WORLD );
```

Comment:
I would say this test is not sufficient - we want to be able to check that the state of the `CommunicationPlan` has not changed either. The operators you have written only duplicate the comm but we will also will need them to copy over the rest of the state (e.g. all of the `std::vector` objects, etc.)

Reply:
I copied inside an existing test instead - I can add more or check each member individually